### PR TITLE
Add dApps prerequisites

### PR DESCRIPTION
**Pull request by andrzej-casper**
### What does this PR introduce?

New dApp page "Prerequisites".

### Additional context

These changes are backported from the default branch of [docs-new](https://github.com/casper-network/docs-new) repository, so you can preview them [live here](https://staging.docs.casper.network/developers/dapps/prerequisites/).

Original PR authored by @dylanireland: https://github.com/casper-network/docs-new/pull/141.

### Checklist

- [x] Docs are successfully building - `yarn install && yarn run build`.
- [x] All technical procedures have been tested.

### CODE REVIEW

**Comment by andrzej-casper** [comment]
This is quite important, because _DevPortal_ homepage is leading to this page, which is under construction:

image

image

File: source/docs/casper/developers/dapps/prerequisites.md [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+
+To ensure the highest level of security, consider the following practices:
+
+1. Code review and auditing: Have your smart contracts and interfaces thoroughly reviewed and audited by experienced professionals. This will help identify any vulnerabilities, bugs, or potential exploits in your code.
+2. Implement best practices: Adhere to industry best practices in smart contract and dApp development. This includes following established design patterns, using secure coding techniques, and staying updated on the latest vulnerabilities and attack vectors.
+3. Testing and simulation: Conduct rigorous testing and simulation of your smart contracts and interfaces. This will help uncover any potential issues or weaknesses before deploying them on the mainnet.
+4. Plan for upgrades and bug fixes: Design your smart contracts to take advantage of Casper Network's support for smart contract upgradability. By doing so, you can ensure that your dApp remains secure and adaptable as the Casper ecosystem evolves, enabling seamless integration of future upgrades and bug fixes.
```
Comment (ACStoneCL) [inline source/docs/casper/developers/dapps/prerequisites.md:28]:
```suggestion
4. Plan for upgrades and bug fixes: Design your smart contracts to take advantage of Casper's support for smart contract upgradability. By doing so, you can ensure that your dApp remains secure and adaptable as the Casper ecosystem evolves, enabling seamless integration of future upgrades and bug fixes.
```

**Review by ACStoneCL — CHANGES_REQUESTED** [review]

**Comment by Adrian-Wrona** [comment]
I changed the link on the DevPortal yesterday. I'll bring it back once this is approved

**Comment by dylanireland** [comment]
Grammatical changes approved. Requesting final review @ACStoneCL

**Review by ACStoneCL — APPROVED** [review]
Issues have been addressed, LGTM.

**Pull request merged**